tests/testArgumentParser.py

```diff
@@ -511,19 +517,16 @@ def testOutputs(self):
 
         # Unspecified
         with self.assertRaises(SystemExit):
-            parser.parse_args(config=self.config, args=[DataPath,])
+            parser.parse_args(config=self.config, args=[DataPath, ])
 
 
-def suite():
-    utilsTests.init()
+class MyMemoryTestCase(utilsTests.MemoryTestCase):
+    pass
 
-    suites = []
-    suites += unittest.makeSuite(ArgumentParserTestCase)
-    suites += unittest.makeSuite(utilsTests.MemoryTestCase)
-    return unittest.TestSuite(suites)
 
-def run(shouldExit = False):
-    utilsTests.run(suite(), shouldExit)
+def setup_module(module):
+    utilsTests.init()
 
-if __name__ == '__main__':
-    run(True)
+if __name__ == "__main__":
+    utilsTests.init()
```

Comment:
Woohoo! (camelCase namespace aliases are a [pet peeve of mine](https://jira.lsstcorp.org/browse/RFC-34)).